Formula/emacs-plus@30.rb

```diff
@@ -158,6 +159,7 @@ def install
 
     args << "--with-modules"
     args << "--with-rsvg"
+    args << "--with-webp" if build.with? "webp"
```

Comment:
I mean, maybe it's better to avoid introducing a new option and instead just build with webp support unconditionally.

```suggestion
    args << "--with-webp"
```

Reply:
Thanks for the heads up, you're right, it would be better to have emacs 30 support webp by default.
I will modify it.

Reply:
I have modified it, please check it, thanks.